src/commands/createContainerApp/EnableIngressStep.ts

```diff
@@ -23,7 +23,8 @@ export class EnableIngressStep extends AzureWizardPromptStep<ICreateContainerApp
     public async getSubWizard(context: ICreateContainerAppContext): Promise<IWizardOptions<ICreateContainerAppContext> | undefined> {
         if (context.enableIngress) {
             return {
-                promptSteps: [new IngressVisibilityStep(), new TargetPortStep()]
+                // @ts-ignore
```

Comment:
The context types are temporarily incompatible until all the ingress commands are decoupled, just doing `@ts-ignore` for now, will remove when `enableIngress` is migrated

Reply:
Maybe add a todo comment and that explanation?